packages/google_maps_flutter/google_maps_flutter/CHANGELOG.md

```diff
@@ -1,338 +1,340 @@
+## 0.5.25+1
+
+- Avoid unnecessary map elements updates by ignoring not platform related attributes (eg. onTap)
```

Comment:
please follow the same pattern in upstream with * instead of -

Reply:
Must be my vscode auto-formatter, my bad, fixed 😉